README.md

```diff
@@ -12,7 +12,24 @@ AWS Lambda is a compute service that allows users to run code, in response to ev
+We put the Lambda functions to read from a particular AWS service (e.g CloudWatch Logs and S3) under each specific folder. Each folder may then have its own instructions to setup the functions.
+
+## Collection Solutions
+| FunctionName | Description | Collection Use Cases | Setup Documentation
+| -------------| ----------- | -------------- | ------------------- |
+|*SumoLogic Lambda Function for AWS CloudWatch Logs With Dead Letter Queue Support*| This project comes with Cloudformation template and two lambda functions which sends CloudWatch logs to Sumo Logic HTTP source endpoint.The first function(invoked by CloudWatch) is configured with DLQ and the second function(invoked periodically by CloudWatch Events) reads from DLQ.| [AWS Lambda ULM App](https://help.sumologic.com/Send-Data/Applications-and-Other-Data-Sources/AWS_Lambda_ULM/Collect_Logs_and_Metrics_for_AWS_Lambda_ULM) | [Docs](https://help.sumologic.com/Send-Data/Collect-from-Other-Data-Sources/Amazon-CloudWatch-Logs)|
+|*SumoLogic Function for AWS CloudWatch Events* | This function is invoked by AWS CloudWatch events in response to state change in your AWS resources which matches a event target definition. The event payload received is then forwarded to Sumo Logic HTTP source endpoint. | [AWS GuardDuty App](https://help.sumologic.com/Send-Data/Applications-and-Other-Data-Sources/Amazon-GuardDuty/Collect-Amazon-GuardDuty-Log-Files) | [Docs](https://github.com/SumoLogic/sumologic-aws-lambda/blob/master/cloudwatchevents/README.md) |
+|*SumoLogic Function for Amazon Inspector* | This function subscribes to a SNS topic where Amazon Inspector publishes its findings.It receives the message payload as an input parameter, transforms it and sends it to Sumo Logic HTTP source endpoint| [Amazon Inspector](https://help.sumologic.com/Send-Data/Applications-and-Other-Data-Sources/Amazon-Inspector-App/) | [Docs](https://help.sumologic.com/Send-Data/Applications-and-Other-Data-Sources/Amazon-Inspector-App/01-Collect-Data-for-Amazon-Inspector) |
+|*Kinesis to Sumo Logic*| This function is invoked by AWS Lambda after it detects new records in Kinesis stream. The received collection of events are decompressed, transformed and send to Sumo Logic HTTP source endpoint |  | [Docs](https://github.com/SumoLogic/sumologic-aws-lambda/blob/master/kinesis/README.md#lambda-configuration) |
+|*SumoLogic Lambda Function for AWS CloudWatch Logs*| This functions subscribes to CloudWatch Log Group and is invoked by AWS CloudWatch with log messages as payload. The records received are decompressed, transformed and  forwarded to Sumo Logic HTTP source endpoint in chunks. | Not Recommended | [Docs](https://help.sumologic.com/Send-Data/Collect-from-Other-Data-Sources/Create-an-Amazon-Lambda-Function) |
```

Comment:
"This functions"  to "This function". Also add: "while the function is more simple then the DLQ-based solution above, it doesn't handle failures and retries properly, thus not recommended.